test/e2e-common.sh

```diff
@@ -138,8 +138,7 @@ function install_pipeline_crd() {
         latestreleaseyaml=https://storage.googleapis.com/tekton-releases-nightly/pipeline/latest/release.yaml
 
     # If for whatever reason the nightly release wasnt there (nightly ci failure?), try the released version
-    [[ -z ${latestreleaseyaml} ]] && \
-        latestreleaseyaml=$(curl -s https://api.github.com/repos/tektoncd/pipeline/releases|python -c "import sys, json;x=json.load(sys.stdin);ass=x[0]['assets'];print([ x['browser_download_url'] for x in ass if x['name'] == 'release.yaml'][0])")
+    [[ -n ${latestreleaseyaml} ]] || latestreleaseyaml="https://storage.googleapis.com/tekton-releases/latest/release.yaml"
```

Comment:
Should this be the following?

```suggestion
    [[ -n ${latestreleaseyaml} ]] || latestreleaseyaml="https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml"
```

`https://storage.googleapis.com/tekton-releases/latest/release.yaml` installs version v0.10.1.

Reply:
you are absoluty right, thanks for spotting this